### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ElminaIusifova**
closes #1576

### CODE REVIEW

File: _pages/en/home.md [hunk 1]
```diff
@@ -6,12 +6,22 @@ lang: en
...
 
-As per the notice that can be found throughout our site, the content you will find all around should be considered **work in progress**: there are no guarantees that the ideas, the proposals and even the presentations that we are developing here in the open will make it into production one day.
+Here, we share our draft strategies, blogs, progress, tools, information, and other resources to support ESDC's digital transformation.
+We create multiple potential solutions, look at the situation from multiple perspectives,  express our professional opinions and ideas when diagnosing  problems.
+Our work is guided by understanding user needs, behaviours, pain points, and building for a specific user segment.
 
-You can have a look at our **[Working Documents]({{ site.baseurl }}/documents-and-presentations.html)**.
+On our website we formulate **[strategies](strategies.md)** and design **[initiatives](initiatives.md)**.
+They will be reviewed and updated at least quarterly based on progress and effectiveness.
+Before you review our work you might want to see **[team's mission](mandate.md)**; where we go as a team and what is our goals.
```
Comment (RachelMuston) [inline _pages/en/home.md:17]: suggestion for minor rewording:  add 'our' before **[teams' mission].

Comment (RachelMuston) [reply _pages/en/home.md:17]: Also for the send half of this sentence, a suggestion for a minor rewording" 'where you can learn about our vision and the principles that guide our work'

**Review by RachelMuston — COMMENTED** [review]
I love text on this page. :-) I've added two minor suggestions but its looking awesome!

File: _pages/en/home.md (hunk 1 above)
Comment (ElminaIusifova) [reply _pages/en/home.md:17]: great suggestion will change it. Thanks

**Review by smellems — APPROVED** [review]

**Pull request merged**